### PR TITLE
Update images and documentation for Model Creator

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/partners/sap/sap-data-models.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Do not change the names of entities, attributes, or associations in the Domain M
 
 ### 2.2 Service Root
 
-The module contains a constant whose value is the service root of the OData service. For example, the **GWSAMPLE_BASIC** data model, generated from the SAP Catalog Service using the OData Model Creator for SAP solutions has a Service Root which is the constant GWSAMPLE_BASIC which has the value `https://sapes5.sapdevcenter.com:443/sap/opu/odata/iwbep/GWSAMPLE_BASIC`.
+The module contains a constant whose value is the service root of the OData service. For example, the **GWSAMPLE_BASIC** data model, generated from the SAP Catalog Service using the Model Creator for SAP solutions has a Service Root which is the constant GWSAMPLE_BASIC which has the value `https://sapes5.sapdevcenter.com:443/sap/opu/odata/iwbep/GWSAMPLE_BASIC`.
```
Comment (gauravkrshah) [inline content/partners/sap/sap-data-models.md]: name need to be changed to "Model Creator for SAP Integrations"

Have to be update at every place

File: content/partners/sap/sap-odata-connector.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The OData Connector for SAP solutions is an OData connector written specifically
 
 ![](attachments/sap-odata-connector/appstore-sapodata.png)
 
-You can use the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/) to generate a data model from the metadata of your OData service, exposed from your SAP back-end system, or from APIs in the SAP API Business Hub or the SAP Catalog Service. For more details see [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator).
+You can use the [Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/) to generate a data model from the metadata of your OData service, exposed from your SAP back-end system, or from APIs in the SAP API Business Hub or the SAP Catalog Service. For more details see [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator).
```
Comment (gauravkrshah) [inline content/partners/sap/sap-odata-connector.md]: link url - https://sapmodelcreator.mendixcloud.com/

have to be done at every place

File: content/partners/sap/use-sap-odata-model-creator.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: "Use OData Model Creator for SAP Solutions"
 category: "SAP"
 menu_order: 35
-description: "Presents the use of the OData Model Creator for SAP solutions."
+description: "Presents the use of the OData functions of the Model Creator for SAP solutions."
```
Comment (gauravkrshah) [inline content/partners/sap/use-sap-odata-model-creator.md]: Change Description to like:
Describes how to use Model Creator to generate/create Mendix model for SAP OData service

File: content/partners/sap/use-sap-odata-model-creator.md [hunk 4]
```diff
@@ -31,18 +31,20 @@ Before starting this how-to, make sure you have completed the following prerequi
 * Obtain authentication credentials, if needed, for the SAP Catalog Service or SAP back-end system you wish to use
 
 {{% alert type="warning" %}}
-OData Model Creator for SAP solutions will produce a module which is in Mendix version 8.15.1. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.15.1 or above.
+The Model Creator for SAP solutions will produce a module which is in Mendix version 8.15.1. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.15.1 or above.
```
Comment (gauravkrshah) [inline content/partners/sap/use-sap-odata-model-creator.md]: Mendix version is 8.18.10

**Review by gauravkrshah — CHANGES_REQUESTED** [review]
Please update

File: content/partners/sap/use-sap-odata-model-creator.md (hunk 4 above)
Comment (MarkvanMents) [reply content/partners/sap/use-sap-odata-model-creator.md]: Done

File: content/partners/sap/sap-data-models.md (hunk 1 above)
Comment (MarkvanMents) [reply content/partners/sap/sap-data-models.md]: Done

File: content/partners/sap/sap-odata-connector.md (hunk 2 above)
Comment (MarkvanMents) [reply content/partners/sap/sap-odata-connector.md]: Done

File: content/partners/sap/use-sap-odata-model-creator.md (hunk 3 above)
Comment (MarkvanMents) [reply content/partners/sap/use-sap-odata-model-creator.md]: Done

File: content/en/docs/partners/sap/sap-data-models.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Mendix apps can consume OData services exposed by SAP back-end systems. To do th
 
 The OData Connector for SAP solutions needs to know details of the OData service which has been exposed. This involves mapping the data structures exposed by the OData service (via the $metadata URL) to entities and attributes of the domain model. This information is created as a Mendix module (.mpk) file which can be imported into your Mendix app.
 
-Any OData service can be used to generate a data model using the OData Model Creator for SAP solutions. See [How To Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator/) for instructions on how to do this.
+Any OData service can be used to generate a data model using the Model Creator for SAP solutions. See [How To Use the Model Creator for SAP Integrations](/partners/sap/use-sap-odata-model-creator/) for instructions on how to do this.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-data-models.md]: Model Creator for SAP Integrations

File: content/en/docs/partners/sap/sap-data-models.md [hunk 6]
```diff
@@ -36,7 +36,7 @@ Do not change the names of entities, attributes, or associations in the Domain M
 
 ### 2.2 Service Root
 
-The module contains a constant whose value is the service root of the OData service. For example, the **GWSAMPLE_BASIC** data model, generated from the SAP Catalog Service using the OData Model Creator for SAP solutions has a Service Root which is the constant GWSAMPLE_BASIC which has the value `https://sapes5.sapdevcenter.com:443/sap/opu/odata/iwbep/GWSAMPLE_BASIC`.
+The module contains a constant whose value is the service root of the OData service. For example, the **GWSAMPLE_BASIC** data model, generated from the SAP Catalog Service using the Model Creator for SAP solutions has a Service Root which is the constant GWSAMPLE_BASIC which has the value `https://sapes5.sapdevcenter.com:443/sap/opu/odata/iwbep/GWSAMPLE_BASIC`.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-data-models.md]: Model Creator for SAP Integrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ The OData Connector for SAP solutions is an OData connector written specifically
 
 {{< figure src="/attachments/partners/sap/sap-odata-connector/appstore-sapodata.png" >}}
 
-You can use the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/) to generate a data model from the metadata of your OData service, exposed from your SAP back-end system, or from APIs in the SAP API Business Hub or the SAP Catalog Service. For more details see [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator/).
+You can use the [Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/) to generate a data model from the metadata of your OData service, exposed from your SAP back-end system, or from APIs in the SAP API Business Hub or the SAP Catalog Service. For more details see [How to Use the Model Creator for SAP Integrations](/partners/sap/use-sap-odata-model-creator/).
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Integrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 8]
```diff
@@ -64,10 +64,10 @@ For more information on domain models, see [Domain Model](/refguide/domain-model
...
 
-You can create a data model by inspecting the service metadata. The response from the service can be used in the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/) to generate a domain model which can be imported into your app. Instructions for doing this are in [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator/).
+You can create a data model by inspecting the service metadata. The response from the service can be used in the [Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/) to generate a domain model which can be imported into your app. Instructions for doing this are in [How to Use the Model Creator for SAP Integrations](/partners/sap/use-sap-odata-model-creator/).
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: Model Creator For SAP Integrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 9]
```diff
@@ -82,15 +82,15 @@ The examples used in this guide are based on the **GWSAMPLE_BASIC** OData servic
 You will need to have an SAP account with access to the [SAP NetWeaver (ES5)](https://sapes5.sapdevcenter.com/) system to be able to use this service.
 {{% /alert %}}
 
-This data model for the GWSAMPLE_BASIC service can be obtained from the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/). To create the data model, you do one of the following:
+This data model for the GWSAMPLE_BASIC service can be obtained from the [Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/). To create the data model, you do one of the following:
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Integrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 10]
```diff
@@ -82,15 +82,15 @@ The examples used in this guide are based on the **GWSAMPLE_BASIC** OData servic
...
 
 * You can create it manually using the OData metadata XML for GWSAMPLE_BASIC, which can be found at [https://sapes5.sapdevcenter.com/sap/opu/odata/iwbep/GWSAMPLE_BASIC/$metadata](https://sapes5.sapdevcenter.com/sap/opu/odata/iwbep/GWSAMPLE_BASIC/$metadata)
-* You can find it in the **SAP Catalog Service** of the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/)
+* You can find it in the **SAP Catalog Service** of the [Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/)
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Integrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 11]
```diff
@@ -82,15 +82,15 @@ The examples used in this guide are based on the **GWSAMPLE_BASIC** OData servic
...
     * Sign in to the server *sapes5.sapdevcenter.com* using your SAP ES5 credentials
     * Search for the GWSAMPLE_BASIC API
     * Use the GWSAMPLE_BASIC Schema
 
-In both cases, save the file you download from the **OData Model Creator for SAP solutions** and then, using the instructions in [How to Use OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator/), import it into your app. Make sure that you review the value of the constant containing the URL of the SAP service you are using after you have imported the data model into your app.
+In both cases, save the file you download from the **Model Creator for SAP solutions** and then, using the instructions in [How to Use Model Creator for SAP Integrations](/partners/sap/use-sap-odata-model-creator/), import it into your app. Make sure that you review the value of the constant containing the URL of the SAP service you are using after you have imported the data model into your app.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Intergrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 12]
```diff
@@ -624,9 +624,8 @@ If you have issues using the OData Connector for SAP Solutions, there is a troub
 * [Entities](/refguide/entities/)
 * [How to Use Marketplace Content in Studio Pro](/appstore/general/app-store-content/)
 * [How to Use the OData Connector for SAP Solutions](/partners/sap/use-sap-odata-connector/)
-* [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator/)
 * [SAP Cloud Connector](/partners/sap/sap-cloud-connector/)
 * [SAP Help Portal](https://help.sap.com)
-* [OData Model Creator for SAP Solutions](https://sapodatamodelcreator.mendixcloud.com/)
+* [Model Creator for SAP Solutions](https://sapmodelcreator.mendixcloud.com/)
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Intergrations

Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 13]
```diff
@@ -14,7 +14,7 @@ aliases:
...
 
-Handcrafting a Mendix domain model for these OData services would be a lot of work. The OData Model Creator for SAP solutions automates this process by creating a Mendix module for the selected service containing the Mendix domain model which can be imported in your project. The data model also contains additional information such as the URL of the exposed endpoint, a list of collections in the service, and a list of functions provided by the service.
+Handcrafting a Mendix domain model for these OData services would be a lot of work. The Model Creator for SAP solutions automates this process by creating a Mendix module for the selected service containing the Mendix domain model which can be imported in your project. The data model also contains additional information such as the URL of the exposed endpoint, a list of collections in the service, and a list of functions provided by the service.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 14]
```diff
@@ -28,22 +28,24 @@ There are four ways to create the data model:
 Before starting this how-to, make sure you have completed the following prerequisites:
 
 * Identify the SAP back-end system and OData service you wish to use
-* Create an app in Mendix Studio Pro (version 8.15.1 or above)
+* Create an app in Mendix Studio Pro (version 8.18.10 or above)
 * Obtain authentication credentials, if needed, for the SAP Catalog Service or SAP back-end system you wish to use
 
 {{% alert color="warning" %}}
-OData Model Creator for SAP solutions will produce a module which is in Mendix version 8.15.1. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.15.1 or above.
+The Model Creator for SAP solutions will produce a module which is in Mendix version 8.18.10. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.18.10 or above.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 15]
```diff
@@ -28,22 +28,24 @@ There are four ways to create the data model:
...
+
+{{< figure src="/attachments/partners/sap/use-sap-odata-model-creator/odata-bapi.png" >}}
 
 {{% alert color="info" %}}
-The OData Model Creator for SAP solutions is not currently available in the Marketplace within Studio Pro. It can only be run in a browser.
+The Model Creator for SAP solutions is not currently available in the Marketplace within Studio Pro. It can only be run in a browser.
 {{% /alert %}}
 
-Open the [OData Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/link/odata), where you will be asked how you want to generate your model:
+Open the [Model Creator for SAP solutions - OData](https://sapmodelcreator.mendixcloud.com/link/odata), where you will be asked how you want to generate your model:
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 16]
```diff
@@ -28,22 +28,24 @@ There are four ways to create the data model:
...
 ## 3 Generating a Data Model
 
-The OData Model Creator for SAP solutions is an app in the Marketplace. Search for it in the Mendix Marketplace, or find it here: [OData Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/link/odata).
+The Model Creator for SAP solutions is an app in the Marketplace. Search for it in the Mendix Marketplace, or find it here: [Model Creator for SAP solutions - OData](https://sapmodelcreator.mendixcloud.com/link/odata). If the app gives you a choice between **SAP OData** and **SAP BAPI**, choose **SAP OData**.
+
+{{< figure src="/attachments/partners/sap/use-sap-odata-model-creator/odata-bapi.png" >}}
 
 {{% alert color="info" %}}
-The OData Model Creator for SAP solutions is not currently available in the Marketplace within Studio Pro. It can only be run in a browser.
+The Model Creator for SAP solutions is not currently available in the Marketplace within Studio Pro. It can only be run in a browser.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 17]
```diff
@@ -28,22 +28,24 @@ There are four ways to create the data model:
...
 
 {{% alert color="warning" %}}
-OData Model Creator for SAP solutions will produce a module which is in Mendix version 8.15.1. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.15.1 or above.
+The Model Creator for SAP solutions will produce a module which is in Mendix version 8.18.10. To use the module you will have to create your app in, or upgrade your app to, Mendix version 8.18.10 or above.
 {{% /alert %}}
 
 ## 3 Generating a Data Model
 
-The OData Model Creator for SAP solutions is an app in the Marketplace. Search for it in the Mendix Marketplace, or find it here: [OData Model Creator for SAP solutions](https://sapmodelcreator.mendixcloud.com/link/odata).
+The Model Creator for SAP solutions is an app in the Marketplace. Search for it in the Mendix Marketplace, or find it here: [Model Creator for SAP solutions - OData](https://sapmodelcreator.mendixcloud.com/link/odata). If the app gives you a choice between **SAP OData** and **SAP BAPI**, choose **SAP OData**.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 18]
```diff
@@ -208,13 +210,13 @@ You have now created the module. The [Using the Data Model Module in a Mendix Ap
 
 ### 3.4 Providing the URL to the Metadata{#URL}
 
-You can also generate the data model from the metadata by providing the URL to the OData Model Creator for SAP solutions.
+You can also generate the data model from the metadata by providing the URL to the Model Creator for SAP solutions.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 19]
```diff
@@ -208,13 +210,13 @@ You have now created the module. The [Using the Data Model Module in a Mendix Ap
...
 
 {{% alert color="warning" %}}
 This method does not work where the destination needs authentication. You cannot use it, for example, to generate a data model for a service on the ES5 Gateway Demo.
 {{% /alert %}}
 
-1. Click **URL** on the first page of the OData Model Creator for SAP solutions.
+1. Click **URL** on the OData page of the Model Creator for SAP solutions.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/use-sap-odata-model-creator.md [hunk 20]
```diff
@@ -273,4 +275,4 @@ For more information on how to use the imported data model together with the ODa
 
 * [How to Use Marketplace Content in Studio Pro](/appstore/general/app-store-content/)
 * [How to Use the OData Connector for SAP Solutions](/partners/sap/use-sap-odata-connector/)
-* [OData Model Creator for SAP Solutions](https://sapmodelcreator.mendixcloud.com/link/odata)
+* [Model Creator for SAP Solutions](https://sapmodelcreator.mendixcloud.com/link/odata)
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/use-sap-odata-model-creator.md]: SAP Intergrations

File: content/en/docs/partners/sap/sap-odata-connector.md [hunk 21]
```diff
@@ -422,7 +422,7 @@ Press <kbd>Enter</kbd> to select the highlighted item.
 
 #### 4.1.1 Odata object
 
-This is an object which is based on an entity type which is a specialization of the OdataObject entity in the OData Connector for SAP solutions domain model. These entities are the ones created in the domain models which you can download from the Marketplace or import via the OData Model Creator for SAP solutions. Objects which are not based on a specialization of the OdataObject entity cannot be used here.
+This is an object which is based on an entity type which is a specialization of the OdataObject entity in the OData Connector for SAP solutions domain model. These entities are the ones created in the domain models which you can download from the Marketplace or import via the Model Creator for SAP solutions. Objects which are not based on a specialization of the OdataObject entity cannot be used here.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-odata-connector.md]: How these entities can be downloaded from Marketplace?
I know this is older document, but this line appears to be confusing.

**Review by gauravkrshah — CHANGES_REQUESTED** [review]
Looks Good. Two changes:
1. Most of the places, "Model creator for SAP Solutions" need to be replaced with "Model Creator for SAP Integrations"
2. Images need to be updated as with new release it is Header text is updated to "Model Creator for SAP Integrations"

File: content/en/docs/partners/sap/sap-odata-connector.md (hunk 21 above)
Comment (MarkvanMents) [reply content/en/docs/partners/sap/sap-odata-connector.md]: @gauravkrshah 
There used to be some OData modules pre-made in the Marketplace. I think they were mainly for customers to quickly test SAP.
Looks like there aren't any now, so I'll look at updating this.

**Comment by MarkvanMents** [comment]
Merging
Will use a new PR to update the rest of the images.

**Pull request merged**